packages/app-builder/src/components/Decisions/RulesDetail.tsx

```diff
@@ -1,116 +1,152 @@
-          {rules.map((rule, index) => (
-            <RuleSection
-              key={`rule_${index}`}
-              value={`rule_${index}`}
-              rule={rule}
-            />
-          ))}
+          {rules.map((rule, index) => {
+            const isTriggered = rule.status === 'triggered';
+
+            const title = (
+              <div className="flex grow items-center justify-between">
+                <div className="text-s flex items-center gap-2 font-semibold">
+                  {rule.name}
+                  {isTriggered ? <Score score={rule.scoreModifier} /> : null}
+                </div>
+                {rule.status ? (
+                  <Tag
+                    border="square"
+                    size="big"
+                    color={isTriggered ? 'green' : 'red'}
+                    className="capitalize"
+                  >
+                    {t(`decisions:rules.status.${rule.status}`)}
+                  </Tag>
+                ) : null}
+              </div>
+            );
+
+            return (
+              <Accordion.Item
+                key={`rule_${index}`}
+                value={`rule_${index}`}
+                className="border-grey-10 overflow-hidden rounded border"
+              >
+                <Accordion.Title className="flex flex-1 items-center justify-between gap-4 p-4">
+                  {title}
+                  <Accordion.Arrow />
+                </Accordion.Title>
+                <Accordion.Content className="bg-purple-02 border-grey-10 border-t">
+                  <div className="flex flex-col gap-4 p-4">
+                    {rule.description ? (
+                      <Callout variant="outlined">{rule.description}</Callout>
+                    ) : null}
+                    <div className="bg-purple-10 inline-flex h-8 w-fit items-center justify-center whitespace-pre rounded px-2 font-normal text-purple-100">
+                      <Trans
+                        t={t}
+                        i18nKey="scenarios:rules.consequence.score_modifier"
+                        components={{
+                          Score: <span className="font-semibold" />,
+                        }}
+                        values={{
+                          score: formatNumber(rule.scoreModifier, {
+                            language,
+                            signDisplay: 'always',
+                          }),
+                        }}
+                      />
+                    </div>
+                    {rule.formula ? (
+                      <RuleFormula
+                        formula={rule.formula}
+                        databaseAccessors={databaseAccessors}
+                        payloadAccessors={payloadAccessors}
+                        astOperators={astOperators}
+                        dataModel={dataModel}
+                        customLists={customLists}
+                        triggerObjectType={triggerObjectType}
+                      />
```

Comment:
This is the key part of this PR: add `<RuleFormula />` for each rule exec on a decision